testing/test_doctest.py

```diff
@@ -117,12 +117,12 @@ def test_simple_doctestfile(self, pytester: Pytester):
     def test_importmode(self, pytester: Pytester):
```

Comment:
This test needed to be updated because now we consider namespace packages correctly. 

Previously it was working by accident due to how we generated our module names, but this as written was not valid in production: `python -c "import namespacepkg.innerpkg"` would not work.